package.json

```diff
@@ -25,6 +25,7 @@
     "@luma.gl/core": "^8.5.10",
     "@probe.gl/stats-widget": "^3.5.0",
     "deck.gl": "^8.8.0-alpha.2",
+    "normalize.css": "^8.0.1",
```

Comment:
What the purpose we need it?

Reply:
It allows us to remove predefined browser styles